lib/config.ml

```diff
@@ -81,46 +107,72 @@ module Ciphers = struct
 
   let fs = fs_of default
 
+  let psk_of = List.filter Ciphersuite.ciphersuite_psk
+
+  let default13 = [
+    `TLS_AES_128_GCM_SHA256 ;
+    `TLS_AES_256_GCM_SHA384 ;
+    (* `TLS_CHACHA20_POLY1305_SHA256 ; *)
+    `TLS_AES_128_CCM_SHA256 ;
+    (* `TLS_AES_128_CCM_8_SHA256 *)
+  ]
+
+  let supported13 = default13
+
 end
 
-let default_hashes =
-  [ `SHA512 ; `SHA384 ; `SHA256 ; `SHA224 ; `SHA1 ]
+(* TODO split into <=12 and >=13, the SHA1 isn't 13 anymore *)
```

Comment:
Relevant entry in the spec seems to be in 4.2.3, page 44:
https://www.rfc-editor.org/rfc/rfc8446.html#page-44
(They differentiate between PKCS1+SHA1 used in certificates vs in the handshake)